apps/system/js/bluetooth_transfer.js

```diff
@@ -82,9 +93,10 @@ var BluetoothTransfer = {
       // https://www.bluetooth.org/en-us/specification/assigned-numbers/
       // service-discovery
       var serviceUuid = '0x1105';
-      var req = adapter.getConnectedDevices(serviceUuid);
+      var req = this._adapter.getConnectedDevices(serviceUuid);
       req.onsuccess = function bt_gotConnectedDevices() {
```

Comment:
Optional suggestion: We can use syntax `=>` instead of `bind`. 
`req.onsuccess = () => {
   // TODO: what we have done
};`